docs/reference-implementation/feature-map.md

```diff
@@ -0,0 +1,252 @@
+
+# EUDI Wallet Reference Implementation - Feature Map
+
+Below you can find an overview of the user functionality provided by the EUDI Wallet Reference Implementation, along with links to relevant functional and technical documentation. 
+
+## Standards, Protocols and Formats
+
+The EUDI Wallet is supported by a defined set of standards and technical specifications considered essential for its implementation. These form the minimum foundation needed to ensure secure issuance, presentation, authentication, signing, and certification of digital identity credentials.
+
+❗For a comprehensive set of standards and technical specifications (STS), you may visit the [Essential Standards and Technical Specifications (STS)](https://eu-digital-identity-wallet.github.io/eudi-doc-architecture-and-reference-framework/latest/technical-specifications/){:target="_blank"} section of the ARF.
+
+In relation to credential formats, the following table indicates the formats supported by the EUDI Wallet Reference Implementation. 
+
+| Format | Status | Description | 
+| --- | --- | --- |
+| `mso_mdoc` | Published (ISO/IEC 18013-5) | Credential format based on ISO/IEC 18013-5 using a signed Mobile Security Object (MSO) to bind and protect data elements. Supports selective disclosure. | 
+| `SD-JWT-VC` | Draft (IETF / OpenID Foundation) | Credential format extending JSON Web Tokens with selective disclosure for web-native verifiable credentials. | 
+
+
+
+
+## Features
+
+The following table provides a high-level overview of the key user functionalities supported by the EUDI Wallet Reference Implementation.
+
+| Features | Description | Status | 
```

Comment:
All these look like in page navigation. Let's be sure that we keep these also sticky in the right sidebar as a table of content